puppet/modules/kubernetes_addons/templates/cluster-autoscaler-deployment.yaml.erb

```diff
@@ -73,9 +73,12 @@ spec:
 <% if @_enable_overprovisioning -%>
         - --expendable-pods-priority-cutoff=-10
 <% end -%>
+<% if @_enable_scale_down_utilization_threshold -%>
```

Comment:
I think you can just do `<% if @scale_down_utilization_threshold -%>`, there is no need for the extra variable. `undef` will return false while any other value will return true.